Update hooks-overview.md (#3110)
reactjs/reactjs.org: content/docs/hooks-overview.md

}
```

> The state of these components are completely independent. Hooks are a way to reuse *stateful logic*, not state itself. In fact, each *call* to a Hook has a completely isolated state -- so you can even use the same custom Hook twice in one component.

Reviewer: The noun here is "state" which is singular so we should use "is", but I agree that this is awkward.

What do you think about this:

```suggestion
The state of each component is completely independent. Hooks are a way to reuse *stateful logic*, not state itself. In fact, each *call* to a Hook has a completely isolated state -- so you can even use the same custom Hook twice in one component.
```

Author: I agree, the latter is much more proper.
